protocol/avdl/keybase1/gui_file_access.avdl

```diff
@@ -0,0 +1,129 @@
+@namespace("keybase.1")
+
+protocol GuiFileAccess {
+
+  import idl "common.avdl";
+
+/*
+  File manipulation interface
+  
+  Caller provides OpID to asynchronous method call, repeats until status is no
+  longer pending or operation is cancelled.
+ 
+  KBFS paths are expected to start with "private" or "public" - mount name is implied.
+*/
+
+  // Random GUID provided by caller 
+  fixed OpID(16);
+  
+  enum ItemType {
+    FILE,
+    DIR
+  }
+
+  record Item {
+    string path;
+    Time time;
+    int size;   // size of file or count of dir entries
```

Comment:
Just FYI -- getting the count of a directory is more expensive that getting the size of the file, it involves an extra FS operation.  That probably doesn't matter to you.   Just letting you know since typically file systems show the directory block size in place of the file size, in things like `ls -l`.  But if the GUI doesn't want to show that and doesn't mind the extra latency, then it seems fine to conflate entry count here.